config/rbac/role.yaml

```diff
@@ -48,6 +48,7 @@ rules:
   resources:
   - persistentvolumeclaims
   verbs:
+  - create
```

Comment:
Role has to be updated in config/dr-cluster/rbac/role.yaml as well.

It is not pretty to have this in 2 places, but for the separate packaging this was done, hence for now need to reflect role changes into the other file as well. (not required for the hub/rbac/role.yaml as PVCs roles are dr-cluster specific)